fe/fe-core/src/main/java/com/starrocks/common/util/TimeUtils.java

```diff
@@ -335,4 +337,23 @@ public static long getNextValidTimeSecond(long startTimeSecond, long targetTimeS
         long step = difference / intervalSecond + 1;
         return startTimeSecond + step * intervalSecond;
     }
+
+    public static PeriodDuration parseHumanReadablePeriodOrDuration(String text) {
+        try {
+            return PeriodDuration.of(PeriodStyle.LONG.parse(text));
+        } catch (DateTimeParseException ignored) {
+            return PeriodDuration.of(DurationStyle.LONG.parse(text));
```

Comment:
what happens if text is "2 years 1 hour"?

Reply:
Will throw exception